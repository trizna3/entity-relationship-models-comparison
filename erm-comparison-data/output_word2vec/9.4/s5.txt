vehicle-vehicle
vehicle type-EMPTY
stop-stops
line-lines
CHANGE_CARDINALITY:S-route-lines
CHANGE_CARDINALITY:S-route-stops
CHANGE_CARDINALITY:S-stops-route
REBIND_1NN1_TO_MN:S-route
