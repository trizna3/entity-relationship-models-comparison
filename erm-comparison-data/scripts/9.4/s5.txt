entity set stops {
	"name"
};

entity set lines {
	"number"
};

entity set route {
	"duration"
};

entity set vehicle {
	"numerical code"
};

entity set timestamps {
};

association between {
	stops ONE
	timestamps ONE
}{};

association between {
	vehicle ONE
	timestamps MANY
}{};

association between {
	lines ONE
	timestamps ONE
}{};

association "schedule" between {
	vehicle MANY
	lines MANY
}{};

association between {
	stops MANY
	route ONE
}{};

association between {
	lines ONE
	route ONE
}{};
