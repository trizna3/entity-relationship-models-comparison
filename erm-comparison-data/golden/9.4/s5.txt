vehicle-vehicle
vehicle type-EMPTY
line-lines
stop-stops